### ISSUE TITLE
Export console messages into a text file available to connected devices

### ISSUE THREAD

**Issue by nesdnuma** [Arksine/moonraker#335]
### Is your feature request related to a problem? Please describe

The idea is to export the text from Mainsail's console but this seem to require some adaptation to Moonraker too.

Please have a look to this feature request in Mainsail: [Export the console messages (commands + results) into a text file](https://github.com/mainsail-crew/mainsail/issues/409):

"Hello,

It would be nice to be able to export the text of the console, for instance when performing some commands with the help of some person who is remote.

Also, it would be nice if this text gathered the commands launched from several devices. So having this saving feature from the RPI side rather than from the client side would be even better (I often launch some commands from my computer and some from my phone)."

Thank you for considering this.

### Describe the solution you'd like

See description of issue.

### Describe alternatives you've considered

----

**Comment by The-Monkey-King** [comment]
Yes, I would like to see this feature added. I am trying to troubleshoot an issue with the Creality K1 Max where I get a system error response appearing in the console but I don't see it in the log files. The only current recourse is to screen shot the message. If I had the console messages saved to a file, I could review both this file and the log files for tracing issues side by side.

The best method is to have this as an inherent method that can be turned off or on (preset to off), with "on" writing the console output to a text file. The file would contain: time stamp / console message. There would be no need to output time stamps with no associated messages.
